From 1f579ea18f4b38a1d3f68700ddc0fd26fa431e8d Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Tue, 8 Aug 2017 16:48:53 +0800
Subject: [PATCH] Revert "sched/tune: Initialize raw_spin_lock in
 boosted_groups"

This reverts commit 9a9259a78c5c723db2a5daaae322f929012d967a.

It seems schedtune_init_cgroups and schedtune_boostgroup_init
all call raw_spin_lock_init(&bg->lock), it is wrong.

Change-Id: Icbdfeaf81f4fb59fdcc02623ac5e26d41bd1e496
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 kernel/sched/tune.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/kernel/sched/tune.c b/kernel/sched/tune.c
index 86d04caf1684..d444fc1a4d58 100644
--- a/kernel/sched/tune.c
+++ b/kernel/sched/tune.c
@@ -647,7 +647,6 @@ schedtune_boostgroup_init(struct schedtune *st)
 		bg = &per_cpu(cpu_boost_groups, cpu);
 		bg->group[st->idx].boost = 0;
 		bg->group[st->idx].tasks = 0;
-		raw_spin_lock_init(&bg->lock);
 	}
 
 	return 0;
-- 
2.35.3

